src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -637,7 +646,7 @@ internal async Task<Collection<FunctionDescriptor>> GetFunctionDescriptorsAsync(
             Collection<FunctionDescriptor> functionDescriptors = new Collection<FunctionDescriptor>();
             var httpFunctions = new Dictionary<string, HttpTriggerAttribute>();
 
-            if (!_environment.IsPlaceholderModeEnabled())
+            if (!_environment.IsPlaceholderModeEnabled() && _httpInvokerOptions == null)
```

Comment:
This code sort of says "if we have http invoker options, we're using a custom worker" but maybe you should make it more explicit, say with a bool property called UseCustomWorker or similar?

Reply:
Here is the sample host.json
```
{
	"version": "2.0",
	"httpInvoker": {
		"description": {
			"defaultExecutablePath": "helloworldServer.exe"
		}
	}
}
```
Do you recommend adding an explicit bool option as well instead of relying on httpInvoker section alone?

Reply:
thanks for sharing this host.json, that clarifies things for me. I think this is probably OK as is. I'll leave a comment in your other PR that covers this config file.